tests/SystemTests.cs

```diff
@@ -68,6 +68,13 @@ await testContext.AssertPullRequests(
                 - Package: Workleap.Extensions.Mongo
                   Type: nuget
                   Update: minor
+            - Title: chore(deps): update dependency workleap.extensions.mongo to redacted
```

Comment:
Too bad our expected test states depend on external factors like us publishing new major library versions (although this shouldn't happen quite often)